qa/tasks/ceph_salt.py

```diff
@@ -61,15 +61,44 @@ class CephSalt(Task):
                         using zypper.
         branch:         Ceph-salt branch in case repo is provided. If no branch
                         is provided master is used by default.
+        containers:
+            registry:
+                name: registry name
+                location: <registry url>
+                insecure: True/False    (Is registry insecure, True by default)
+            image:  <container image path>
```

Comment:
maybe call this `image_ceph`? I think there are more images (eg. one for monitoring)

Reply:
done, I renamed it to `ceph_image`